platforms/avr/drivers/i2c_master.c

```diff
@@ -293,7 +311,9 @@ i2c_status_t i2c_readReg16(uint8_t devaddr, uint16_t regaddr, uint8_t* data, uin
     return (status < 0) ? status : I2C_STATUS_SUCCESS;
 }
 
-void i2c_stop(void) {
-    // transmit STOP condition
-    TWCR = (1 << TWINT) | (1 << TWEN) | (1 << TWSTO);
-}
+__attribute__((weak)) i2c_status_t i2c_ping_address(uint8_t address, uint16_t timeout) {
+    i2c_status_t status;
+    status = i2c_start(address, timeout);
```

Comment:
```suggestion
    i2c_status_t status = i2c_start(address, timeout);
```

Just a nit to make the code smaller.